packages/PhpAttribute/Printer/PhpAttributeGroupFactory.php

```diff
@@ -45,6 +42,8 @@ public function __construct(
         // nested indexes supported only since PHP 8.1
         if (! $phpVersionProvider->isAtLeastPhpVersion(PhpVersionFeature::NEW_INITIALIZERS)) {
             $this->unwrappedAnnotations['Doctrine\ORM\Mapping\Table'][] = 'indexes';
+            $this->unwrappedAnnotations['Doctrine\ORM\Mapping\Table'][] = 'uniqueConstraints';
+            $this->unwrappedAnnotations['Doctrine\ORM\Mapping\Entity'][] = 'uniqueConstraints';
```

Comment:
You can tweak the phpstan notice with single assign

Reply:
Fixed!